README.md

```diff
@@ -121,4 +121,20 @@ vendor/bin/roave-backward-compatibility-check --help
 
 ## Configuration
 
-There are currently no configuration options available.
+The file `.roave-backward-compatibility-check.xml` is read from the current working directory (when it exists) and sets configuration for the command.
+
+It's expected to be an XML file that follows our [schema](resources/schema.xsd):
+
+**Example:**
+
+```xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<roave-bc-check
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="resources/schema.xsd">
```

Comment:
I imagine this would be `vendor/roave/backward-compatibility-check/resources/schema.xsd` for a consumer, I'd think?

Reply:
Yup

Reply:
It depends™️ 😬 

If this lib has been installed using composer, yes.
If this lib has been installed using phar, not really... maybe the full URL would be better.

I was conflicted when I wrote that one...

Reply:
The documentation example should likely assume `vendor/` path: `.phar` is really someone else's problem 😁